docker/portal-demo/README.md

```diff
@@ -0,0 +1,86 @@
+ICGC DCC - Portal Demo
+===
+
+ICGC DCC Portal Demo is a docker container that allows to configure and run a standalone version of the [ICGC DCC Portal.](https://dcc.icgc.org/)
```

Comment:
Please mention how this is for demo purposes only and not intended for production environments.